src/templates/contentPage.js

```diff
@@ -67,10 +113,83 @@ class ContentPage extends Component {
   buildPageHTML(pageContentHTML) {
     return { __html: pageContentHTML };
   }
+  
+  //using the page list, create the grid of cards
+  mapPageListDataToElem() {
+    let row = [], 
+    pageListGrid = [],
+    max = 2,
+    titleCard,
+	themePalette;
+
+    // list needs to be in grid, max 2 per row
+    for (let i = 0; i < max; i+=2) {
+      row = [];
+	  
+	  titleCard = this.props.data.contentfulPage.previousPage;
+	  if (titleCard !== null){
+		themePalette = this.themePaletteToCss(this.props.data.contentfulPage.previousPage.parentMenu.themePalette)
+		row.push(this.pageItemToCard(titleCard, themePalette));
+	  }
+	  
+	  titleCard = this.props.data.contentfulPage.nextPage;
+	  if (titleCard !== null){
+		themePalette = this.themePaletteToCss(this.props.data.contentfulPage.nextPage.parentMenu.themePalette)
+	    row.push(this.pageItemToCard(titleCard, themePalette));
+	  }
+
+      pageListGrid.push(
+        <div className="columns">
+          { row.map(col => { return(<div className="column">{col}</div>) }) }
+        </div>
+      );
+    }
+
+    return pageListGrid;
+  }
+  
+  //generate a card from a page
+  pageItemToCard(page, themePalette) {
+    const url = `/${page.parentMenu.slug}/${page.slug}`;
+    const title = this.getCardTitleForLanguage(page.titleList, this.state.language);
+
+    return (
+      <div className={`content ${themePalette.cardParentContainerHover}`}>
+        <div className={`card ${cardSize} ${themePalette.background}`}>
+          <div className={`card-content ${cardTitleText} ${vertCentred}`}>
+              <Link className={`title ${themePalette.text}`} to={url}>
```

Comment:
Again, you should be able to theme these using the scss. I have all the colors in the main scss file you can pull for 'is-healthcare' etc etc